src/index.html

```diff
@@ -5,9 +5,62 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+      >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
```

Comment:
It is a bad practice to connect the SCSS file directly in HTML file.
Please compile your SCSS into a CSS file, and link the CSS file in your HTML instead.

Reply:
it was written in the task description:
https://github.com/FrontKid/layout_product-cards/tree/develop#the-task

if i compiled in css there will be no spaces between the lines, I don't know how to set it up in the plugin